src/Root.cpp

```diff
@@ -151,6 +151,11 @@ void cRoot::Start(std::unique_ptr<cSettingsRepositoryInterface> a_OverridesRepo)
 		LOGWARN("Regenerating settings.ini, all settings will be reset");
 		IniFile->AddHeaderComment(" This is the main server configuration");
 		IniFile->AddHeaderComment(" Most of the settings here can be configured using the webadmin interface, if enabled in webadmin.ini");
+
+		IniFile->AddKeyName("Server");
+		IniFile->AddKeyComment("Server", " MaxLoadedChunks specifies how many chunks should be loaded at the same time");
```

Comment:
We need a standarized method for INI comments. But for now, I just put this here because I think it's important for players to configure this option.